app.py

```diff
@@ -0,0 +1,74 @@
+import flask
```

Comment:
File "/home/user67/CitiesGame/GameManager.py", line 17, in load_games_info
    for file in os.listdir(path=path):
FileNotFoundError: [Errno 2] No such file or directory: '/home/user67/CitiesGame/games'